src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+'Hello, world!'
+</body>
+</html>
```

Comment:
```suggestion
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
  </head>
  <body>
    Hello, world!
  </body>
</html>
```

Reply:
tests failed because of incorrect indents, here is how it should be like
there is a simple rule – 1 nested tag – +2 indents as compared to its parent tag
```
<A> -- +0
  <B> -- +2
    <C> -- +4
      <D> -- +6
        ...
      </D>
    </C>
  </B>
</A>
```